src/common/StatementMetadata.cpp

```diff
@@ -167,7 +167,7 @@ const char* StatementMetadata::getPlan(bool detailed)
 
 	if (plan->isEmpty())
 	{
-		UCHAR info[] = {detailed ? isc_info_sql_explain_plan : isc_info_sql_get_plan};
```

Comment:
/Users/nakagami/firebird/src/common/StatementMetadata.cpp:170:19: error:
          non-constant-expression cannot be narrowed from type 'unsigned int' to
          'UCHAR' (aka 'unsigned char') in initializer list [-Wc++11-narrowing]
      ...info[] = {detailed ? isc_info_sql_explain_plan : isc_info_sql_get_plan};
                   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~